pom.xml

```diff
@@ -15,6 +15,11 @@
     
     <repositories>
         <repository>
+          <id>central</id>
+          <name>central</name>
+          <url>https://repo1.maven.org/maven2</url>
+        </repository>
+        <repository>
```

Comment:
Yeah, maybe I'm not guessing right about what went wrong here and whether this would fix it.  Reacting to [this failed build](https://travis-ci.org/apetro/hrs-portlets/builds/46898851).